features/netsocket/cellular/generic_modem_driver/TESTS/unit_tests/default/template_mbed_app.txt

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
check the docs here https://github.com/ARMmbed/mbed_OS_API_Docs/blob/5.5/docs/APIs/communication/cellular.md#port-verification-testing

You can't have a json file here. Yes, it is template file, whose contents are copied to a json file when it is desirable to run these particular tests. A tutorial is also part of the file, check lines after license.